CASMTRIAGE 6749: DOCS (#4848)
Cray-HPE/docs-csm: operations/iuf/workflows/management_rollout.md

        --no-config-change --config-name "${CFS_CONFIG_NAME}" --xnames $MASTER_STORAGE_XNAMES --clear-state
        ```

>         The sample output for a management node is:

Reviewer: Suggesting to change this line since it is configuring multiple management nodes.
```suggestion
       Sample output for configuring multiple management nodes is:
```

Author: made the change
